notebooks/04_nexus_displaying_data.ipynb

```diff
@@ -252,265 +243,417 @@
        "      <td>https://bbp.epfl.ch/nexus/v1/resources/nse/tes...</td>\n",
        "      <td>https://bbp.epfl.ch/nexus/v1/resources/nse/tes...</td>\n",
        "      <td>https://bbp.epfl.ch/nexus/v1/resources/nse/tes...</td>\n",
+       "      <td>https://bbp.epfl.ch/nexus/v1/resources/nse/tes...</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>type</th>\n",
        "      <td>DetailedCircuit</td>\n",
        "      <td>DetailedCircuit</td>\n",
        "      <td>DetailedCircuit</td>\n",
        "      <td>DetailedCircuit</td>\n",
+       "      <td>DetailedCircuit</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>brainLocation.brainRegion.id</th>\n",
        "      <td>http://api.brain-map.org/api/v2/data/Structure...</td>\n",
        "      <td>http://api.brain-map.org/api/v2/data/Structure...</td>\n",
        "      <td>http://api.brain-map.org/api/v2/data/Structure...</td>\n",
        "      <td>http://api.brain-map.org/api/v2/data/Structure...</td>\n",
+       "      <td>http://api.brain-map.org/api/v2/data/Structure...</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>brainLocation.brainRegion.label</th>\n",
        "      <td>Thalamus</td>\n",
        "      <td>Thalamus</td>\n",
        "      <td>Thalamus</td>\n",
        "      <td>Thalamus</td>\n",
+       "      <td>Thalamus</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>circuitBase.type</th>\n",
        "      <td>DataDownload</td>\n",
        "      <td>DataDownload</td>\n",
        "      <td>DataDownload</td>\n",
        "      <td>DataDownload</td>\n",
+       "      <td>NaN</td>\n",
```

Comment:
Are all these NaNs in the dataframes expected?

Reply:
Yeah, it's the default pandas' behavior if the data is incomplete. 
I have a mention in the notebook of this just before this cell but unfortunately it's a bit difficult to see it in the diff.